inlong-agent/agent-common/src/main/java/org/apache/inlong/agent/db/TaskProfileDb.java

```diff
@@ -63,7 +62,7 @@ public void storeTask(TaskProfile task) {
         if (task.allRequiredKeyExist()) {
             String keyName = getKeyByTaskId(task.getTaskId());
             KeyValueEntity entity = new KeyValueEntity(keyName,
-                    task.toJsonStr(), task.get(TaskConstants.FILE_DIR_FILTER_PATTERNS));
+                    task.toJsonStr(), "");
```

Comment:
why put an empty value here?

Reply:
For task configuration, it is not possible to fill in specific files here, and we do not need file names as indexes